package.json

```diff
@@ -5,8 +5,8 @@
   "repository": "https://github.com/coinbase/onchainkit.git",
   "license": "MIT",
   "scripts": {
-    "build": "packemon build --addEngines --addFiles --declaration && npx packemon validate --no-license --no-people --no-repo && tailwindcss -i ./src/index.css -o ./src/styles.css --minify",
-    "watch": "packemon watch",
+    "build": "packemon build --addEngines --addFiles --declaration && npx packemon validate --no-license --no-people --no-repo && tailwindcss -i ./src/index.css -o ./src/styles.css --postcss --minify",
```

Comment:
what postcss does?

Reply:
this will allow us to import files at the root index.css

Reply:
actually... might not need postcss. Let me play around with it

Reply:
ok

Reply:
We don't need postcss and just use native css import. Removed postcss